app/constants/import.js

```diff
@@ -1,5 +1,10 @@
 const robotImportUser = { username: 'robot-import-system-user', displayname: 'Robot Import' }
 
+const overnightJobUser = {
```

Comment:
Can I suggest this be moved to its own constants file (or be part of another constants file). Although this file already has a user, the overnight user doesn't relate to the import. And we will eventually remove the import process (including this file)